infra/bisector.py

```diff
@@ -65,29 +48,30 @@ def main():
   parser = argparse.ArgumentParser(
       description='git bisection for finding introduction of bugs')
 
-  parser.add_argument(
-      '--project_name',
-      help='The name of the project where the bug occured',
-      required=True)
-  parser.add_argument(
-      '--commit_new',
-      help='The newest commit SHA to be bisected',
-      required=True)
-  parser.add_argument(
-      '--commit_old',
-      help='The oldest commit SHA to be bisected',
-      required=True)
-  parser.add_argument(
-      '--fuzz_target', help='the name of the fuzzer to be built', required=True)
+  parser.add_argument('--project_name',
+                      help='The name of the project where the bug occured',
```

Comment:
Being consistent is important. Since other help messages for parser (like in `detect_repo.py`) are ended with periods this should be as well.

Reply:
Done.